deploy/tests/network.ts

```diff
@@ -623,131 +627,322 @@ import { getNamedAccounts } from 'hardhat';
+                                .addLiquidityETH(
+                                    token.address,
+                                    NATIVE_TOKEN_DESIRED_AMOUNT,
+                                    1,
+                                    1,
+                                    linkWhale.address,
+                                    MAX_UINT256,
+                                    {
+                                        value: NATIVE_TOKEN_DESIRED_AMOUNT
+                                    }
+                                );
+
+                            const poolTokenAddress = await factory.getPair(weth, link);
+                            poolToken = await Contracts.ERC20.attach(poolTokenAddress);
+
+                            await poolToken.connect(linkWhale).approve(bancorPortal.address, poolTokenAmount);
                         });
 
-                        expect(await getBalance(nativeToken, masterVault.address)).to.be.almostEqual(
-                            prevVaultTokenBalance.add(nativeTokenAmount),
-                            {
-                                maxRelativeError: new Decimal('0.001')
+                        it('should perform portal migrations', async () => {
+                            const { liquidity: prevLinkLiquidity } = await poolCollection.poolData(link);
+                            const { liquidity: prevNativeTokenLiquidity } = await poolCollection.poolData(
+                                NATIVE_TOKEN_ADDRESS
+                            );
+
+                            switch (type) {
+                                case PoolType.UniswapV2: {
+                                    await bancorPortal
+                                        .connect(linkWhale)
+                                        .migrateUniswapV2Position(weth, link, poolTokenAmount);
+                                    break;
+                                }
+
+                                case PoolType.SushiSwapV1: {
+                                    await bancorPortal
+                                        .connect(linkWhale)
+                                        .migrateSushiSwapV1Position(weth, link, poolTokenAmount);
+                                    break;
+                                }
+
+                                default:
+                                    throw new Error(`Unsupported type: ${type}`);
                             }
-                        );
+
+                            const { liquidity: newLinkLiquidity } = await poolCollection.poolData(link);
+
+                            expect(newLinkLiquidity.stakedBalance).to.be.almostEqual(
+                                prevLinkLiquidity.stakedBalance.add(depositedLinkAmount),
+                                {
+                                    maxRelativeError: new Decimal('0.0000000000000000001')
+                                }
+                            );
+                            expect({
```

Comment:
trading liquidity might change if the pool doesn't have excess liquidity, maybe it's enough testing for staked amounts?

Reply:
sounds good